articles/azure-functions/durable/durable-functions-concepts.md

```diff
@@ -34,6 +34,25 @@ In the function chaining pattern, a sequence of functions executes in a specific
 
 You can use Durable Functions to implement the function chaining pattern concisely as shown in the following example:
 
+#### Precompiled C#
+
+```csharp
+public static async Task<object> Run([OrchestrationTrigger]DurableOrchestrationContext context)
```

Comment:
Nit: Let's make sure there's a space between the attribute and parameter it's attached to, as in our samples, for consistency.

This comment applies to any other instance of the same in this document.